### PR TITLE
Renamed package.json to dub.json and added sourcePaths

### PR DESCRIPTION
**Pull request by BlackEdder**
Needed to actually compile with dub.

Also added dub build files to .gitignore

### CODE REVIEW

File: dub.json [hunk 1]
```diff
@@ -6,5 +6,6 @@
 	"copyright": "Copyright (c) 2015 Manu Evans",
 	"license": "Boost",
 	"importPaths": ["."],
-	"targetType": "sourceLibrary"
+	"targetType": "sourceLibrary",
+    "sourcePaths": ["."]
```
Comment (TurkeyMan) [inline dub.json]: Added spaces to a file with tabs >_<

**Comment by TurkeyMan** [comment]
Thanks for the fix. Would you mind amending the spaces? :)

**Comment by BlackEdder** [comment]
Will do so tomorrow.

**Comment by BlackEdder** [comment]
Amended

**Pull request merged**